ursula-inspec/mysql/controls/mysql.rb

```diff
@@ -0,0 +1,94 @@
+control 'MYSQL001' do
+  impact 0.5
+  title 'Does a mysql user exist?'
+  desc 'Check if mysql user exists since the mysql process should be run my a not-root user'
+  tag 'production','development'
+  tag 'mysql'
+  tag remediation: 'ursula <env> site.yml --tags=mysql'
+  ref 'https://dev.mysql.com/doc/refman/5.7/en/security-guidelines.html'
+  describe user('mysql') do
+    it { should exist }
+  end
+end
+
+control 'MYSQL002' do
+  impact 0.5
+  title 'Check if mysqld process is running'
+  desc 'Mysqld process should be run my a non-root user.'
+  tag 'production','development'
+  tag 'mysql'
+  tag remediation: 'ursula <env> site.yml --tags=mysql'
+  ref 'https://dev.mysql.com/doc/mysql-security-excerpt/5.7/en/security-against-attack.html'
+  describe processes('mysqld') do
```

Comment:
we need to find a way to say "If mysqld process is running, it should be running as mysql user"  ...  because its not a compliance issues if the process isn't running.